dao/src/main/java/greencity/repository/BagRepository.java

```diff
@@ -67,13 +67,25 @@ public interface BagRepository extends JpaRepository<Bag, Integer> {
         + "WHERE obm.ORDER_ID = :orderId", nativeQuery = true)
     List<Map<String, Object>> getBagInfo(Long orderId);
 
+    /**
+     * method, that returns {@link List}of{@link Bag}.
+     *
+     * @param orderId order id
+     * @return {@link List}of{@link Bag} by orderId.
+     * @author Max Boiarchuk
+     */
+    @Query(nativeQuery = true,
+        value = "SELECT * FROM order_bag_mapping AS obm JOIN bag AS b ON obm.bag_id = b.id "
+            + "WHERE obm.order_id = :orderId")
+    List<Bag> findAllByOrder(@Param("orderId") Long orderId);
+
     /**
      * method, that returns {@link List} of {@link Bag}'s that matches by
      * {@link TariffsInfo} and {@link MinAmountOfBag}.
      *
      * @param tariffId       order id {@link TariffsInfo}
      * @param minAmountOfBag order id {@link MinAmountOfBag}
-     * @author Oleg Vatuliak
+     * @author José Castellanos
```

Comment:
wrong place
you should change this line https://github.com/ita-social-projects/GreenCityUBS/pull/935/files#diff-994757fde39b95b6b6b79c13c61aab0a12cc0b0b4a75cee76f487764ba19f0cbR75